src/Microsoft.DotNet.Migration/project.json

```diff
@@ -0,0 +1,72 @@
+{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "warningsAsErrors": true
+  },
+  "dependencies": {
+    "dotnet": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "NuGet.Commands": {
+      "version": "3.5.0-rc1-1697",
+      "exclude": "compile"
+    },
+    "NuGet.CommandLine.XPlat": "3.5.0-rc1-1697",
+    "Newtonsoft.Json": "9.0.1",
+    "System.Text.Encoding.CodePages": "4.0.1",
+    "System.Diagnostics.FileVersionInfo": "4.0.0",
+    "Microsoft.ApplicationInsights": "2.0.0",
```

Comment:
Is this all copied from 'dotnet'?  Since you already reference 'dotnet', you should be able to remove most of these.